packages/node-loader/lib/index.js

```diff
@@ -15,12 +16,39 @@ import {createFormatAwareProcessors} from '@mdx-js/mdx/lib/util/create-format-aw
 export function createLoader(options) {
   const {extnames, process} = createFormatAwareProcessors(options)
 
-  return {getFormat, transformSource}
+  return {load, getFormat, transformSource}
 
+  /* c8 ignore start */
+  // Node version 17.
+  /**
+   * @param {string} url
+   * @param {unknown} context
+   * @param {Function} defaultLoad
```

Comment:
Does Node have a more specific type for the default loader?

Reply:
The API for `defaultLoad` is the same as this `load` function.

Reply:
Could we create a recursive interface for the type?

Reply:
Sure but I don’t see a bunch of value. It just got redesigned. It’ll be redesigned again. The API is unstable: https://nodejs.org/api/esm.html#loadurl-context-defaultload